src/content/docs/style-guide/writing-docs/writer-workflow/process-changes.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Proposing process or workflow changes
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+Use this document to learn how to propose changes to the process or workflow in a collaborative manner.
+
+## Proposing a change
+
+Do you have an idea you want to try out? It can be something big or something little or even something you haven't figured out how to solve yet. It can be a small suggestion such as using conventional commits, or a large scale change such as automating PR reviews. If so, follow these steps so the team can collaborate on the issue or suggestion.
+
+1. Write your idea as a GitHub issue. The issue should have a brief overview, the actual proposal, and any relevant action items. See [this issue](https://github.com/newrelic/docs-website/issues/4553) for an example. If all you have is an issue and no proposal, that is fine too. The team can collaborate to solve it in the next step.
```

Comment:
```suggestion
1. Write your idea as a GitHub issue. The issue should have a brief overview, the actual proposal, and any relevant action items. See [this issue](https://github.com/newrelic/docs-website/issues/4553) for an example. If all you have is an issue and no proposal, that's fine too. The team can collaborate to solve it in the next step.
```

Reply:
Style guide says to avoid things like "this issue" as the link text to avoid any suspicion of click bait. https://docs.newrelic.com/docs/style-guide/formatting/hyperlinks/  Better practice is to tell the reader exactly where the link goes.